packages/clay-css/src/images/icons/flags-en-CA.svg

```diff
@@ -0,0 +1,13 @@
+<!--
+* SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2022 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
+
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We shouldn't use `width` or `height` attributes in the SVG. This prevents the icon from scaling when we change the `font-size`. We should use `viewBox="0 0 16 16"` instead.